tests/ReflectorTest.php

```diff
@@ -210,11 +212,34 @@ private function createReflector(string $code): Reflector
 
         $sourceLocator = new AggregateSourceLocator([
             new StringSourceLocator($code, $astLocator),
-            new PhpInternalSourceLocator($astLocator),
+            new PhpInternalSourceLocator(
```

Comment:
this is duplicated in several places, so maybe this should be refactored accordingly

Reply:
You can simplify it with `BetterReflection::sourceStubber()`

Reply:
The configuration is a bit different though